playbooks/files/export-dcn.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+# Script to execute on the central site to generate useful
+# data to deploy edge sites.
+
+unset OS_CLOUD
+export OS_AUTH_TYPE=none
+export OS_ENDPOINT=http://127.0.0.1:8006/v1/admin
+DIR=/home/stack/exported-data
+
+mkdir -p $DIR
+openstack stack output show standalone EndpointMap --format json \
+| jq '{"parameter_defaults": {"EndpointMapOverride": .output_value}}' \
+> $DIR/endpoint-map.json
+
+openstack stack output show standalone HostsEntry -f json \
```

Comment:
How about using the long format option, `--format` for consistency?

Reply:
sure, I'll do it